frog/imports/client/Preview/ConfigPanel.jsx

```diff
@@ -130,7 +130,7 @@ class ConfigPanel extends React.Component<*, *> {
       setActivityTypeId
     } = this.props;
     if (
-      metadatas.owner_id === Meteor.user().username &&
+      metadatas.owner_id === getUsername() &&
```

Comment:
All of this logic is even worse now that multiple users can have the same display name - you don't even need a separate server to impersonate another user :) But that's not your problem, we'll have to rethink this completely at some point, but we'll leave it for now. Anyway you can't access another users sessions and student data, only the publicly stored graph.